static/js/components/UserMenu.js

```diff
@@ -25,10 +25,41 @@ export const DropUpArrow = () => (
   <i className="material-icons arrow_drop_up">arrow_drop_up</i>
 )
 
+export const LoggedInMenu = (props: Object) => (
+  <DropdownMenu {...props}>
```

Comment:
by typing the `props` here as `Object` we lose any typechecking that `DropdownMenu` is receiving the props it expects - maybe just make a `MenuProps` object like this for typing these props (and in the `LoggedOutMenu` as well)?

```js
type MenuProps = {
  className: "string",
  closeMenu: Function,
}
```